_build/Extension-ReplaceNames.ps1

```diff
@@ -19,10 +19,13 @@ Param(
   [string]$commandCategory,
 
   [Parameter(Mandatory = $True, Position = 7)]
-  [string]$wizardLaunchCommand,
+  [string]$createWebAppCommand,
```

Comment:
Question: where and when should we add the deployReactNativeCommand? Probably when we implement that part?

Reply:
though for web, not native apps. Not in the near future.